CHANGELOG.md

```diff
@@ -16,6 +16,17 @@ changes in the following format: PR #1234***
 - Addition of `PhaseEncodingDirection` and `EchoNumber` columns to the `files`, `files_qcstatus`
   and `feedback_mri_comments` tables to ensure uniqueness keys for specific GE sequences for
   which the `SeriesUID/EchoTime` combination is not enough (PR #8152).
+  
+### Modules
+#### API 
+- Use PSCID instead of the CandID in the candidates API (PR #8138)
+
+
+#### Bug Fixes  
+- Fix ConfigurationException bug (PR #8107)
+- Fix PHP fatal error when running the LORIS installation script (PR #8108)
+- Add psr/log to composer (PR #8109)
```

Comment:
I don't think the above 3 need to be mentioned because I think they were just broken on the branch and not the latest release, but "Add support for PHP 8.1" should probably be mentioned

Reply:
@driusan removed!

Reply:
oops. missed one part of your comment. Added PHP 8.1 support in last commit.